__tests__/user/user.query.test.js

```diff
@@ -4,24 +4,58 @@ const client = require('../../utils/apollo-test-client');
 const {
```

Comment:
Why do we need lines `#2, #3` if we are not using client anymore?

Reply:
We use gql variables in requests to the server.